src/DumbPasswordServiceProvider.php

```diff
@@ -28,12 +35,13 @@ class DumbPasswordServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        $path = realpath(__DIR__.'/../resources/config/passwordlist.txt');
-        $dumbPasswords = collect(explode("\n", file_get_contents($path)));
-        $data = $dumbPasswords->flip();
-
-        Validator::extend('dumbpwd', function ($attribute, $value, $parameters, $validator) use ($data) {
-            return !$data->has($value);
+         Validator::extend('dumbpwd', function ($attribute, $value, $parameters, $validator) {
+            $path = realpath(__DIR__ . '/../resources/config/passwordlist.txt');
+            $cache_key = md5_file($path);
+            $data = Cache::rememberForever('dumbpwd_list_' . $cache_key, function () use ($path) {
```

Comment:
oops. i had added it in the project I tested this in, but not the fresh clone of the repo. fix incoming!